src/openstudio_lib/ModelObjectListView.hpp

```diff
@@ -60,15 +62,20 @@ class ModelObjectListController : public OSVectorController
  private:
   openstudio::IddObjectType m_iddObjectType;
   model::Model m_model;
-  bool m_showLocalBCL;
+  bool m_isLibrary;
```

Comment:
`m_showLocalBCL` meant to show LocalBCL objects, basically just meant that this was a library controller, I thought `m_isLibrary` was more clear.